localstack-core/localstack/services/resourcegroupstaggingapi/tagging/plugin.py

```diff
@@ -0,0 +1,71 @@
+import logging
+
+from plux import Plugin, PluginManager
+
+from localstack.aws.api import RequestContext
+from localstack.utils.objects import singleton_factory
+
+LOG = logging.getLogger(__name__)
+
+TAGGING_PLUGIN_NAMESPACE = "localstack.service.tagging.plugins"
+
+
+class TaggingPlugin(Plugin):
+    """
+    The plugin is to be used to expose the inherited service plugins to the Resource Groups Tagging API.
+    This will allow supported service resources to be tagged via this API by calling the individual service
+    implementations of `tag_<resource>` or `untag_<resource>` whilst obeying IAM restrictions.
+    """
+
+    namespace = TAGGING_PLUGIN_NAMESPACE
+
+    def tag_resource(
+        self, context: RequestContext, resource_arn: str, tags: dict[str, str]
+    ) -> None:
```

Comment:
Even though these operations are idempotent from RTG provider's perspective, it may be useful to return a value that indicates the operation status.

For example, for untag_resource operation, returning the number of tag_keys that were removed.